spot-client/webpack.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
         compress: true,
         contentBase: path.join(__dirname, '/'),
         historyApiFallback: true,
+        host: mode === 'development' ? '0.0.0.0' : 'localhost',
```

Comment:
Is the ternary necessary? I'm wondering when the mode wouldn't be development. Do the webdriver tests still pass? They might be defaulted to localhost.

Reply:
Yeah probably it's not, at first it was not there, but then I have spotted the mode and that made me think it can be only in dev.

I will check on the tests

Reply:
but that devServer is used only in dev mode anyway ? is that it ?

Reply:
`npm run test` passes

Reply:
It's used only for launching the dev server, correct. The mode is used to for webpack to do extra work, like uglifying when it is set to production.

Reply:
> `npm run test` passes

I meant the tests in spot-webdriver/, not the unit tests in spot-client/.

Reply:
K, I checked, looks like. I clearly don't understand how this "host" field works.

Reply:
No it's not working for localhost now...

Reply:
I am about to close it. I don't know if this worked with the ternary, but will try now if not will close the PR

Reply:
Are you sure you want to close? I think it's fine as long as we still have a convenient way to run the server and tests locally.

Reply:
I would have to find a way to direct the test to 0.0.0.0 instead of localhost

Reply:
The tests default to localhost; they don't require it. Look for this line:
`const TEST_SERVER_URL = process.env.TEST_SERVER_URL || 'http://localhost:8000';`

Reply:
oh thanks, I couldn't find that

Reply:
hmm I think I'm just dump, I didn't realized that webpack must be started manually for the tests

Reply:
yes, the tests are working as is

Reply:
That is correct. We do not have a script that starts the server and runs the tests--something we'll need to re-enable jenkins testing. Right now the tests take in whatever URL and run against that.

Reply:
yes, that makes sense